_posts/2016-08-06-release-add-ons-1.9.md

```diff
@@ -0,0 +1,37 @@
+---
+title:  "Release: Add-Ons 1.9"
+---
+
+We are proud to announce the availability of Add-On Administration 1.9, Add-On Self Administration 1.9,
+and Add-On Self Administration Plug-In API 1.6.
+These releases are mainly fixing bugs, but there's also a new feature in the Add-On Administration.
+This release officially deprecates the support for OSIAM 2.x, which will be removed in version 1.12 or 2.0.
+Furthermore, this will probably be the last release of all add-ons, as they will be merged into the core server soon.
```

Comment:
There will be also no non-standalone release. What I mean here, is that there will be no further releases whatsoever. Functionality will be re-implemented in the core.